uiowa/uiowa
# [CCOM] CCOM logo theme settings

Resolves https://github.com/uiowa/uiowa/issues/6629

# How to test

`ddev blt frontend`

## Test update hook on a regents theme
```
ddev blt ds --site=wirrc.org && ddev drush @org-wirrc.local uli /admin/appearance/settings/uids_base
```

1. Update your local.settings `$config['stage_file_proxy.settings']['origin'] = 'https://wirrc.org';`
2. Confirm that update hook sets the branding option to "Regent" and the color is still set to "gray"
3. Inspect source and confirm that custom favicon paths for the site still exist
4. Inspect page title org and confirm that it is still "Board of Regents"

## Test on default

```
ddev blt ds --site=default && ddev drush @default.local uli /admin/appearance/settings/uids_base
```

1. Confirm that update hook sets the branding option to "Iowa" and the color is still set to "brand"
2. Confirm that block Iowa favicons are still rendering
3. Inspect page title or and verify that it is still "The University of Iowa"

## Test CCOM 


```
ddev blt ds --site=default && ddev drush @default.local config-set uids_base.settings header.branding_options ccom -y &&  ddev drush @default.local config-set uids_base.settings style.style_selector brand -y && ddev drush @default.local uli /admin/appearance/settings/uids_base
```

1. Confirm that new logo is rendering in place of block Iowa in the header and footer
2. Switch Site name display to "Below Iowa bar" at `/admin/appearance/settings/uids_base` and confirm that new logo appears in small tab
4. Confirm that favicon dome renders in place of the block Iowa favicon
5. Inspect page title org and confirm that it is "Page title | Site name | Carver college of medicine"
